### ISSUE TITLE
Softmax activations don't get converted to Softmax TFLite operator if ndim > 2

### ISSUE THREAD

**Issue by ghop02** [tensorflow/tensorflow#35199]
**System information**
- OS Platform and Distribution (e.g., Linux Ubuntu 16.04): MacOS 10.15.1
- TensorFlow installed from (source or binary): binary
- TensorFlow version (or github SHA if from source): tf-nightly==2.1.0.dev20191203

**Command used to run the converter or code if you’re using the Python API**

```
import pathlib

inpt = tf.keras.layers.Input(shape=[256, 256, 3])
out = tf.keras.layers.Lambda(lambda x: tf.keras.activations.softmax(x))(inpt)
out = tf.keras.layers.Lambda(lambda x: tf.nn.softmax(x))(out)
model = tf.keras.Model(inpt, out)

converter = tf.lite.TFLiteConverter.from_keras_model(model)
tflite_model = converter.convert()
pathlib.Path('out.tflite').write_bytes(tflite_model)
```

**Failure details**
![image](https://user-images.githubusercontent.com/1422280/71025023-3904e880-20d4-11ea-95fb-29cfea49a44d.png)
This graph shows the difference between the different softmax methods.  When using `tf.keras.activations.softmax`, there is [code](https://github.com/tensorflow/tensorflow/blob/v2.1.0-rc1/tensorflow/python/keras/activations.py#L43-L79) with a workaround for multiple dimensions.  It looks like this was written before the tensorflow op had multi-dimension support.

**Comment by jvishnuvardhan** [comment]
@ghop02 Can you please try `experimental_new_converter` and let us know whether the issue persists with most recent converter. I had tried the converter for several complex layers and the converter is working well. I have added `experimental_new_converter` to your code. Please check the [gist here](https://colab.sandbox.google.com/gist/jvishnuvardhan/85eebf3e3df8471d9b87b70943b759d0/untitled715.ipynb). 

`converter.experimental_new_converter = True`

Thanks!

**Comment by ghop02** [comment]
Hi @jvishnuvardhan , thanks for the response!

![image](https://user-images.githubusercontent.com/1422280/71234090-acba1780-22c5-11ea-8f33-1ea7fc12a051.png)

I still get a similar result.  If you look at the [link above](https://github.com/tensorflow/tensorflow/blob/v2.1.0-rc1/tensorflow/python/keras/activations.py#L43-L79), the keras softmax is converting to:
```
  elif ndim > 2:
    e = math_ops.exp(x - math_ops.reduce_max(x, axis=axis, keepdims=True))
    s = math_ops.reduce_sum(e, axis=axis, keepdims=True)
    return e / s
```

I think removing that logic, and just always returning the tf.nn.softmax operation would work, since I believe that code was written before tensorflow had support for `ndim > 2`

**Comment by jdduke** [comment]
Seems reasonable though I don't have the full history behind the Keras implementation. @fchollet any thoughts (or somebody else who can chime in)?

**Comment by ghop02** [comment]
Any additional thoughts on this? Just bumping in case it got lost during the holidays!

**Comment by anilsathyan7** [comment]
Facing same issue with tf.keras(TF 2.1.0), when is use x=Activation("softmax")(ip).
The tflite conversion is successful; but the layers appear as shown by @ghop02 

![tfbug](https://user-images.githubusercontent.com/1130185/73448772-9b196480-4387-11ea-82bc-32e6b15e5820.png)

**Comment by ghop02** [comment]
@anilsathyan7 if you're blocked, you should be able to make this change to sneak the softmax operator in:
```diff
- out = Activation("softmax")(in)
+ out = tf.keras.layers.Lambda(lambda x: tf.nn.softmax(x))(in)
```

**Comment by jdduke** [comment]
@ghop2: If you want to submit a PR for updating the [Keras softmax behavior](https://github.com/tensorflow/tensorflow/blob/v2.1.0-rc1/tensorflow/python/keras/activations.py#L73), we'd be happy to approve. Thanks again for flagging the issue.

**Comment by sinclairnick** [comment]
> @anilsathyan7 if you're blocked, you should be able to make this change to sneak the softmax operator in:
> 
> ```diff
> - out = Activation("softmax")(in)
> + out = tf.keras.layers.Lambda(lambda x: tf.nn.softmax(x))(in)
> ```

I will also add that TF2.1 works with conversion but TF2.2 does not, when using this method
